TACC/protx-dashboard
# task/COOKS-246: configure client app for deployment

## Overview: ##

This PR sets up the image to serve the dashboard client 

## Related Jira tickets: ##

* [COOKS-246](https://jira.tacc.utexas.edu/browse/COOKS-246)

## Testing Steps: ##

Best to bring up and test as normal

Run-normally-and-test
1. `docker-compose build`
2. `docker-compose up`
3.  do other README setup steps (i.e. use `manage.py`...`npm run dev`)
4. login
5.  ensure you can get to this dashboard `https://cep.dev/protx/dash/maltreatment`

Test-client-that-is-built-in-container

0. **ensure you had already run docker-compose build since checking out this branch**
1. 'docker-compose down`
2. stop running `npm run dev` as we are checking that the client app is built and works from the image
3. edit docker-compose to change USE_DEV_CLIENT to false (or remove that line)
4. `docker-compose up`
6. ensure you can get to this dashboard `https://cep.dev/protx/dash/maltreatment`

## Notes: ##
- [x] build client in image
- [x] setup template serve SPA from flask
